commit_build.bat

```diff
@@ -1 +1,6 @@
+rem test set: jUnit tests for java and javascript, integration tests
```

Comment:
I would remove the word "set" from the first line of each file. Let's leave only "tests: " and further info.